middle_end/flambda2/to_cmm/to_cmm_static.ml

```diff
@@ -49,6 +49,18 @@ let rec static_block_updates symb env res acc i = function
       in
       static_block_updates symb env res acc (i + 1) r)
 
+let rec static_unboxed_int_array_updates symb env res acc i = function
+  | [] -> env, res, acc
+  | sv :: r -> (
+    match (sv : _ Or_variable.t) with
+    | Const _ -> static_unboxed_int_array_updates symb env res acc (i + 1) r
+    | Var (var, dbg) ->
+      let env, res, acc =
+        C.make_update env res dbg Word_int ~symbol:(C.symbol ~dbg symb) var
```

Comment:
This `Word_int` seems wrong and unprecise (but mostly wrong, e.g. for int32). You should probably take a `Cmm.memory_chunk` as argument of the `static_unboxed_int_array_updates` and pass it instead of `Word_int`.